src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <form action="https://www.google.com/search"
+            class="search-bar"
+            method="get"
+            data-qa="big"
+      >
```

Comment:
все еще неправильное форматирование кода. По ссылке checklist в задании есть примеры правильного форматирования, сделай по примеру
```suggestion
      <form
        action="https://www.google.com/search"
        class="search-bar"
        method="get"
        data-qa="big"
      >
```